frontend/automation/cypress/e2e/network-configuration.cy.js

```diff
@@ -58,20 +55,153 @@ describe('network configuration', () => {
     cy.getByTestId('token-expiry').should('have.text', '168h0m0s')
   })
 
-  it('edit network details displayed', () => {
+  it('edit network details displayed', function () {
```

Comment:
suggestion (non-blocking): i think for consistency, it's better to just stick with the lambdas here

```suggestion
  it('edit network details displayed', () => {
```

Reply:
rectified